services/apis/nondbgapUsersync/nonDbgapTasks.js

```diff
@@ -106,36 +108,29 @@ module.exports = {
     return id;
   },
 
-  async presignedURLRequest(fence, project, projectFile, user, token) {
-    console.log(`Creating presigned url with mainAcct user for ${project} after running usersync`);
-    console.log(`${token}`);
-    const signedURLs3 = await fence.do.createSignedUrl(projectFile, ['protocol=s3'], token);
-    if (signedURLs3.status === 200) {
-      console.log(`The presigned url for ${project} files is created. The S3 url -> ${signedURLs3.data.urls}`);
-    }
-    const signedURLgs = await fence.do.createSignedUrl(projectFile, ['protocol=gs'], token);
-    if (signedURLgs.status === 200) {
-      console.log(`The presigned url for ${project} files is created. The GS url -> ${signedURLgs.data.urls}`);
-    }
-
+  async presignedURLRequestS3(fence, project, projectFile, user, token) {
     let projectFileContents3 = null;
-    let projectFileContentgs = null;
-
-    try {
+    console.log(`Creating presigned url with ${user} user for ${project} after running usersync`);
+    const signedURLs3 = await fence.do.createSignedUrl(projectFile, ['protocol=s3'], token);
+    if (signedURLs3.status !== 200) {
+      console.log(`User ${user} with access can not create s3 signed urls and read the file for ${project}`);
+    } else {
+      console.log(`The presigned url for ${project} files is created. The S3 url -> ${signedURLs3.data.url}`);
       projectFileContents3 = await fence.do.getFileFromSignedUrlRes(signedURLs3);
-    } catch (err) {
-      console.log('Failed to fetch presigned url', err);
+      expect(projectFileContents3).to.equal(fence.props.awsBucketInfo.cdis_presigned_url_test.testdata);
     }
+  },
 
-    try {
+  async presignedURLRequestGs(fence, project, projectFile, user, token) {
+    let projectFileContentgs = null;
+    console.log(`Creating presigned url with ${user} user for ${project} after running usersync`);
+    const signedURLgs = await fence.do.createSignedUrl(projectFile, ['protocol=gs'], token);
+    if (signedURLgs.status !== 200) {
+      console.log(`User ${user} with access can not create gs signed urls and read the file for ${project}`);
```

Comment:
Same as previous comment. Might not always be a 401 error so we should output the request message so its easier to debug